sympy/physics/units/tests/test_unit_system_cgs_gauss.py

```diff
@@ -26,18 +25,47 @@ def test_conversion_to_from_si():
     assert convert_to(joule, erg, SI) == 10**7*erg
     assert convert_to(joule, erg, cgs_gauss) == 10**7*erg
 
+
     assert convert_to(dyne, newton, SI) == newton/10**5
     assert convert_to(dyne, newton, cgs_gauss) == newton/10**5
     assert convert_to(newton, dyne, SI) == 10**5*dyne
     assert convert_to(newton, dyne, cgs_gauss) == 10**5*dyne
 
 
+def test_ohm_cgs_gauss():
+
+    assert convert_to(ohm,second/centimeter,cgs_gauss) == 25000*second/(22468879468420441*centimeter)
+    assert convert_to(1*ohm,second/centimeter,cgs_gauss) == 25000*second/(22468879468420441*centimeter)
+    assert convert_to(2*ohm,second/centimeter,cgs_gauss) == 50000*second/(22468879468420441*centimeter)
+    assert NS(convert_to(ohm,second/centimeter,cgs_gauss)) == '1.11265005605362e-12*second/centimeter'
+    assert NS(convert_to(2*ohm,second/centimeter,cgs_gauss)) == '2.22530011210724e-12*second/centimeter'
+
+def test_henry_cgs_gauss():
+    assert convert_to(henry,second**2/centimeter,cgs_gauss) == 25000*second**2/(22468879468420441*centimeter)
+    assert convert_to(1*henry,second**2/centimeter,cgs_gauss) == 25000*second**2/(22468879468420441*centimeter)
+    assert convert_to(2*henry,second**2/centimeter,cgs_gauss) == 50000*second**2/(22468879468420441*centimeter)
+    assert NS(convert_to(henry,second**2/centimeter,cgs_gauss)) == '1.11265005605362e-12*second**2/centimeter'
+    assert NS(convert_to(2*henry,second**2/centimeter,cgs_gauss)) == '2.22530011210724e-12*second**2/centimeter'
+
+def test_volt_cgs_gauss():
+    assert convert_to(volt,statvolt,cgs_gauss) == 10**6*statvolt/299792458
+    assert convert_to(1*volt,statvolt,cgs_gauss) == 10**6*statvolt/299792458
+    assert convert_to(2*volt,statvolt,cgs_gauss) == 2*10**6*statvolt/299792458
```

Comment:
ditto

Reply:
This test is for the conversion of a `volt` to a `statvolt`
In master when we are converting `volt` to `statvolt` it gives a wrong answer.
So I fix it to `cgs_gauss.set_quantity_scale_factor(volt, 10**6/speed_of_light*statvolt)`
from `cgs_gauss.set_quantity_scale_factor(volt, speed_of_light*statvolt/10**6)`
Reference for the [conversions](https://en.wikipedia.org/wiki/Gaussian_units) of SI unit to gaussian units